Homework 3/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Homework_3
+{
+    class Program
+    {
+        private static void Main(string name, string surname, int age, float weight, float height, float bmi)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'Main'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887828)